### PR TITLE
introduce issue and pull request templates

### PR DESCRIPTION
**Pull request by zouyee**
### 1. Why is this pull request needed and what does it do?

With issue and pull request templates, you can customize and standardize the information you'd like contributors to include when they open issues and pull requests in coredns repository.

### 2. Which issues (if any) are related?
NONE
### 3. Which documentation changes (if any) need to be made?
NONE
### 4. Does this introduce a backward incompatible change or deprecation?
NONE

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+name: Bug Report
+about: Report a bug encountered while using CoreDNS 
+labels: bug
+
+---
+
+<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
+
+If the matter is security related, please disclose it privately via https://kubernetes.io/security/
```
Comment (SuperQ) [inline .github/ISSUE_TEMPLATE/bug-report.md]: Is this the correct security contact info?

Comment (zouyee) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Done

**Review by SuperQ — APPROVED** [review]
LGTM.

**Review by yongtang — APPROVED** [review]
lgtm

**Pull request merged**